web-basics-browser-security-model/activities/cockroach/src/main.template.py

```diff
@@ -0,0 +1,10 @@
+from flask import Flask, request
+app = Flask(__name__)
+
+@app.route('/cockroach', methods=['DELETE'])
+def delete_this_bastard():
+    return "__TEMPLATE__"
+
+
+if __name__ == '__main__':
+    app.run(debug=True, host='127.0.0.1')
```

Comment:
```suggestion
    app.run(host='127.0.0.1')
```
Maybe don't enable debug in "production"